### PR TITLE
*created banditry category

### PR DESCRIPTION
**Pull request by tomklecha**
*slight change in json file - erased object at the beginning

### CODE REVIEW

**Comment by tomklecha** [comment]
@Canato please look my edit of questions.json -> with your version we have error of reading the json - object, instead we require straight away array. So for me easier was to change json. But obviously i should change is more in code, was not sure how. So for now i left it like that

**Comment by Canato** [comment]
> @Canato please look my edit of questions.json -> with your version we have error of reading the json - object, instead we require straight away array. So for me easier was to change json. But obviously i should change is more in code, was not sure how. So for now i left it like that

I see your point and it is a valid solution. But the real solution is to change our DTO. 

If this was a API endpoint you would not be able to twist .

Is important to be a object, cause maybe in the future I want to change the API/Endpoint to have : 
```
{
  "object": [...questions... ],
"timeStamp": 456789039847
}
```

So we have a timestamp from the last this was update. With a object instead of a array we would not break old clients 
Backward compatibility (Let me know if you don't understand the reasons for this) 

**Please undo the changes in the json and fix the code so I can review**

**Comment by tomklecha** [comment]
in commit comment should be revert json changes, my mistake - now its working

File: features/questions/src/main/java/com/can_apps/questions/bresenter/QuestionsModel.kt [hunk 1]
```diff
@@ -22,7 +22,18 @@ internal enum class QuestionIdModelEnum {
...
+    BANDITRY_DRIVE_DRUNK,
+    BANDITRY_LOST_LICENCE,
+    BANDITRY_ACCUSED,
+    BANDITRY_JUVENILE,
+    BANDITRY_STEALING,
+    BANDITRY_SOLD_STOLEN,
+    BANDITRY_NOTION,
+    BANDITRY_PREGNANT,
+    BANDITRY_ABORTION,
+    BANDITRY_BOUGHT_STOLEN
```
Comment (Canato) [inline features/questions/src/main/java/com/can_apps/questions/bresenter/QuestionsModel.kt:36]: 11? 🤔

File: features/questions/src/test/java/com/can_apps/questions/data/mappers/QuestionsCategoryAssetMapperDefaultTest.kt [hunk 2]
```diff
@@ -12,11 +12,12 @@ internal class QuestionsCategoryAssetMapperDefaultTest {
     @Test
     fun `GIVEN category, WHEN asset_mapCategoryToDomain, THEN return QuestionCategoryDomainEnum`() {
         // GIVEN
-        val category = listOf("drugs", "sex", "religion", "pterodactyl")
+        val category = listOf("drugs", "sex", "religion", "banditry", "pterodactyl")
```
Comment (Canato) [inline features/questions/src/test/java/com/can_apps/questions/data/mappers/QuestionsCategoryAssetMapperDefaultTest.kt:15]: `"pterodactyl"` Hehehe I laugh every time XD

**Pull request merged**